### PR TITLE
Base the ztp container on ubi9, not ubi8

### PR DESCRIPTION
**Pull request by lack**
Signed-off-by: Jim Ramsay <jramsay@redhat.com>

### CODE REVIEW

**Comment by lack** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-images

**Comment by sabbir-47** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-images

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ base_images:
   os-min8:
```
Comment (nishant-parekh) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:22]: Update this to os-min9 for the mapping to work.

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml [hunk 2]
```diff
@@ -45,7 +45,7 @@ images:
   to: dpdk
 - context_dir: .
   dockerfile_path: ztp/resource-generator/Containerfile
-  from: os-min8
+  from: os-min9
```
Comment (nishant-parekh) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml:48]: Or use base-9 like the dpdk container above

**Comment by lack** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-images

**Review by nishant-parekh — APPROVED** [review]

**Comment by yuvalk** [comment]
/hold
seems team is not ready for ubi9 yet :-)

**Pull request closed without merging**